Python/syntax_test_python.py

```diff
@@ -47,7 +47,28 @@
 #      ^^ invalid.illegal.unexpected-relative-import.python
 import .. sys
 #      ^^ invalid.illegal.unexpected-relative-import.python
+import *
+#      ^ invalid.illegal.name.import.python
 
+from os import *, path
+#^^^ meta.statement.import.python - meta.import-source - meta.import-path
+#   ^ meta.statement.import.python meta.import-source.python - meta.import-path
+#    ^^ meta.statement.import.python meta.import-source.python meta.import-path.python
+#      ^ meta.statement.import.python meta.import-source.python - meta.import-path
+#       ^^^^^^^^ meta.statement.import.python - meta.import-source - meta.import-path
+#               ^^^^^^^ - meta.statement
+#              ^ constant.language.import-all.python
```

Comment:
Minor nitpick, but this should be one line higher.

Reply:
No, it shouldn't. First set of test cases addresses (primarily) meta scopes while the second chunk addresses "normal" scopes. This has become some kind of common scheme in many test files and may cause something like that.

Normally, those second parts should also start with the first token to be complete, like in line 79, but that's out of scope of this PR, IMHO.